specification/migrate/resource-manager/Microsoft.Migrate/AssessmentProjects/preview/2023-05-01-preview/migrate.json

```diff
@@ -12185,6 +12309,122 @@
           },
           "readOnly": true,
           "x-ms-identifiers": []
+        },
+        "estimatedExternalStorage": {
+          "type": "array",
+          "description": "Estimated External Storage",
+          "items": {
+            "$ref": "#/definitions/EstimatedExternalStorage"
+          },
+          "readOnly": true,
+          "x-ms-identifiers": []
+        },
+        "estimatedNetwork": {
+          "type": "array",
+          "description": "Estimated Network",
+          "items": {
+            "$ref": "#/definitions/EstimatedNetwork"
+          },
+          "readOnly": true,
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "AvsEstimatedExternalStorage": {
+      "type": "object",
+      "description": "Details on the Estimated External Storage for AVS Assessment.",
+      "properties": {
+        "monthlyPrice": {
+          "type": "number",
+          "format": "float",
+          "description": "Total monthly cost for type of storage."
```

Comment:
The currency field is provided by the customer only in PUT request body and We have a currency field in the avsAssessment Resource. It will be of the same currency.